terminologies/technique/probabilityMapping.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/probabilityMapping",

```

Comment:
I like the "probabilistic diffusion tractography". Could we use this similarly here (to be a bit more specific?): "probabilistic cytoarchitectonic mapping"?

@tgbugs: what do you think?